src/v2/Components/Footer.tsx

```diff
@@ -71,7 +71,7 @@ const DownloadAppBanner = () => {
           <Image
             height={320}
             width="102%"
-            src="https://files.artsy.net/consign/banner-large.jpg"
+            src="https://d196wkiy8qx2u5.cloudfront.net/?resize_to=fit&width=957&height=320&quality=70&src=https://files.artsy.net/consign/banner-large.jpg"
```

Comment:
Can you use the [`resized`](https://github.com/artsy/force/blob/master/src/v2/Utils/resized.ts) function instead of doing this manually.

There's some further optimizations that can be made here as well: https://github.com/artsy/force/blob/c587b39c12cfeb8e09ddc142b919460cb7b40c65/src/v2/Apps/Fair/Components/FairHeader/FairHeaderImage.tsx#L29-L36 — though I can follow up with these if you like.

Reply:
Totally!